src/components/search/SearchCard.jsx

```diff
@@ -1,129 +1,45 @@
 import TourItem from './TourItem'
-// import fetchHotel from '../../fetch/search/fetchHotel'
-// import fetchTour from '../../fetch/search/fetchTour'
-
-const hotels = [
-  {
-    id: '1',
-    name: '그랜드워커힐',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-  {
-    id: '2',
-    name: '신라호텔',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-  {
-    id: '3',
-    name: '그랜드워커힐',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-  {
-    id: '4',
-    name: '신라호텔',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-  {
-    id: '5',
-    name: '그랜드워커힐',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-  {
-    id: '6',
-    name: '신라호텔',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-  {
-    id: '7',
-    name: '신라호텔',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-    price: '200,000',
-  },
-]
-
-const tours = [
-  {
-    id: '1',
-    name: '롯데월드',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-  },
-  {
-    id: '2',
-    name: '에버랜드',
-    city: '용인',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-  },
-  {
-    id: '3',
-    name: '에버랜드',
-    city: '용인',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-  },
-  {
-    id: '4',
-    name: '에버랜드',
-    city: '용인',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-  },
-  {
-    id: '5',
-    name: '롯데월드',
-    city: '서울',
-    country: '대한민국',
-    rating: '4.2',
-    reviewCount: '100',
-  },
-]
+import fetchSearchHotel from '../../fetch/fetchSearchHotel'
+import fetchSearchTour from '../../fetch/fetchSearchTour'
 
 function SearchCard() {
   // 검색어 useParams로 가져오기 e.g. 서울, 종로, 일본
   const { keyword } = useParams()
+  // 한 페이지에 불러 올 아이템 수
+  const items = 1
+  const [attractions, setAttractions] = useState([])
+  const [hotels, setHotels] = useState([])
+
+  const hotelRes = useQuery({
+    queryKey: ['hotels', keyword, items],
+    queryFn: fetchSearchHotel,
+  })
+  const attractionRes = useQuery({
+    queryKey: ['attractions', keyword, items],
+    queryFn: fetchSearchTour,
+  })
 
-  // useQuery로 데이터 fetch comments는 테스트용
-  // 종합 결과 페이지에서는 페이지 데이터 x로 1페이지만 불러옴
-  // const hotelList = useQuery(['hotels', keyword], fetchHotel)
-  // const tourList = useQuery(['tours', keyword], fetchTour)
+  // 위의 데이터에서 필요한 값 가져옴
+  const hotelData = hotelRes?.data?.lodging
+  const attractionData = attractionRes?.data?.attraction
+
+  useEffect(() => {
+    if (!hotelRes.isLoading && hotelData) {
+      setHotels(() => [...hotelData])
+    }
+  }, [hotelData, hotelRes.isLoading])
```

Comment:
useEffect와 useState를 같이 쓰는거니 아래와 비슷하는건 않을까요?
```suggestion
  const hotels = useMemo(() => {
    if (hotelRes.isLoading || !hotelData)  return []
    return setHotels(() => [...hotelData])
  }, [hotelData, hotelRes.isLoading])
```

Reply:
아 넵 감사합니다 코치님 그 부분이 잘 이해가 안가서 고치지 못했는데
방금 머지해서 지금 해당 부분들 고쳐놨다가 다음 pr때 반영할 수 있도록 하겠습니다 감사합니다